lib/client/utils.js

```diff
@@ -51,12 +51,12 @@ self = module.exports = {
 
         switch (argType) {
         case "array":
-            if (!arg instanceof Array) {
+            if (!(arg instanceof Array)) {
                 invalidArg = true;
             }
             break;
         case "date":
-            if (!arg instanceof Date) {
+            if (!(arg instanceof Date)) {
```

Comment:
Haha- not sure- `jshint` was reporting it, it seemed. Does it show up for you? (using jshint v2.7.0) Maybe it was a recent addition in a release to catch such things?